aws/dms/tasks.yml

```diff
@@ -55,6 +55,8 @@ cron-pii:
 - dashboard.learning_goal_teacher_evaluations
 - dashboard.learning_goal_ai_evaluation_feedbacks
 - dashboard.ai_tutor_interactions
+- dashboard.aichat_requests
+- dashboard.lti_deployments_user_identities
```

Comment:
```suggestion
- dashboard.lti_deployments_user_identities
remove_column:
- stored_messages
- new_message
```

Reply:
Oh, I think we do want stored_messages/new_message for now. The join would bring in a new column that doesn't exist on any table yet.